### ISSUE TITLE
Crash on put vanilla hive into ceramic hive

### ISSUE THREAD

**Issue by dann1kid** [VinterNacht/FromGoldenCombs#8]
```
Game Version: v1.16.3 (Stable)
Loaded Mods: backpackpack@1.0.3, dragonflies@1.0.8, locustmod@2.3.5, game@1.16.3, carrycapacity@0.6.4, fromgoldencombs@1.3.6-rc.3, medievalexpansion@3.7.1, playercorpse@1.3.0, prospectorinfo@3.1.2, stepup@1.2.0, creative@1.16.3, survival@1.16.3, wildfarming@1.2.3, workbenchexpansion@1.4.1
14.02.2022 17:07:56: Critical error occurred
System.InvalidCastException: Не удалось привести тип объекта "FromGoldenCombs.BlockEntities.FGCBeehive" к типу "FromGoldenCombs.BlockEntities.BECeramicBroodPot".
   в FromGoldenCombs.Blocks.CeramicBroodPot.GetPlacedBlockInteractionHelp(IWorldAccessor world, BlockSelection selection, IPlayer forPlayer)
   в Vintagestory.Client.NoObf.HudElementBlockInteractionHelp.OnBlockChanged(BlockPos pos, Block oldBlock)
   в Vintagestory.API.Client.BlockChangedDelegate.Invoke(BlockPos pos, Block oldBlock)
   в Vintagestory.Client.NoObf.ClientEventManager.TriggerBlockChanged(ClientMain game, BlockPos pos, Block oldBlock)
   в Vintagestory.Client.NoObf.ClientMain.OnPlayerTryPlace(BlockSelection blockSelection, String& failureCode)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnBlockBuild(BlockSelection blockSelection, Block onBlock, String& failureCode)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   в Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   в Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   в Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   в _TcPKe0ld0EAbHR8YsMG1OLy9q8O._vg1QFi5yvtdWtcWdYGzwV9j9PVf(Single )
   в _SbWHHe8Ad6lBV8IRbvWFqut7ren._mSnyaJUIQhBb8A1j3UUn7uh5BBO(Single )
   в _SbWHHe8Ad6lBV8IRbvWFqut7ren._b365Cpnna1p8GSaDyyO3EH2VfOD(Single )
   в Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   в OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 479
   в OpenTK.GameWindow.DispatchRenderFrame() в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 454
   в OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 375
   в _muFH2BhSjNKo8GSc2o33Nm6Ckup._SwLkq72vc0vcc3OnHXTAS19pm9p(_tVmXPdKJ3JBDWFc8tBtvgftcfhl , String[] )
   в _XL1Xa21FKJVFetNmIumId5p6UDU._SwLkq72vc0vcc3OnHXTAS19pm9p(ThreadStart )
-------------------------------
```

**Comment by dann1kid** [comment]
```
-------------------------------
Game Version: v1.16.3 (Stable) by Skif_97
Loaded Mods: backpackpack@1.0.3, dragonflies@1.0.8, locustmod@2.3.5, game@1.16.3, carrycapacity@0.6.4, fromgoldencombs@1.3.6-rc.3, medievalexpansion@3.7.1, playercorpse@1.3.0, creative@1.16.3, survival@1.16.3, wildfarming@1.2.3, workbenchexpansion@1.4.1
15.02.2022 19:44:14: Critical error occurred
System.InvalidCastException: Не удалось привести тип объекта "FromGoldenCombs.BlockEntities.FGCBeehive" к типу "FromGoldenCombs.BlockEntities.BECeramicBroodPot".
   в FromGoldenCombs.Blocks.CeramicBroodPot.GetPlacedBlockInteractionHelp(IWorldAccessor world, BlockSelection selection, IPlayer forPlayer)
   в Vintagestory.Client.NoObf.HudElementBlockInteractionHelp.OnBlockChanged(BlockPos pos, Block oldBlock)
   в Vintagestory.API.Client.BlockChangedDelegate.Invoke(BlockPos pos, Block oldBlock)
   в Vintagestory.Client.NoObf.ClientEventManager.TriggerBlockChanged(ClientMain game, BlockPos pos, Block oldBlock)
   в Vintagestory.Client.NoObf.ClientMain.OnPlayerTryPlace(BlockSelection blockSelection, String& failureCode)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnBlockBuild(BlockSelection blockSelection, Block onBlock, String& failureCode)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   в Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   в Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   в Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   в _TcPKe0ld0EAbHR8YsMG1OLy9q8O._vg1QFi5yvtdWtcWdYGzwV9j9PVf(Single )
   в _SbWHHe8Ad6lBV8IRbvWFqut7ren._mSnyaJUIQhBb8A1j3UUn7uh5BBO(Single )
   в _SbWHHe8Ad6lBV8IRbvWFqut7ren._b365Cpnna1p8GSaDyyO3EH2VfOD(Single )
   в Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   в OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 479
   в OpenTK.GameWindow.DispatchRenderFrame() в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 454
   в OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 375
   в _muFH2BhSjNKo8GSc2o33Nm6Ckup._SwLkq72vc0vcc3OnHXTAS19pm9p(_tVmXPdKJ3JBDWFc8tBtvgftcfhl , String[] )
   в _XL1Xa21FKJVFetNmIumId5p6UDU._SwLkq72vc0vcc3OnHXTAS19pm9p(ThreadStart )
-------------------------------```

**Comment by VinterNacht** [comment]
Thank you for the ongoing reports. These are all the same issue, thankfully.

**Comment by dann1kid** [comment]
> Thank you for the ongoing reports. These are all the same issue, thankfully.
tl dr
different  clients. One's rare, one's frequently

**Comment by VinterNacht** [comment]
Resolved in upcoming release
